cmd/all-in-one/main.go

```diff
@@ -159,20 +163,13 @@ by default uses only in-memory database.`,
 				log.Fatal(err)
 			}
 
-			telset := telemetry.Setting{
-				Logger:         svc.Logger,
-				TracerProvider: tracer.OTEL,
-				Metrics:        queryMetricsFactory,
-				ReportStatus:   telemetry.HCAdapter(svc.HC()),
-				LeveledMeterProvider: func(_ configtelemetry.Level) metric.MeterProvider {
-					return noop.NewMeterProvider()
```

Comment:
should this be replaced with a meter provider? or do we not need to pass anything here because its noop?

Reply:
I added noop to L100, but afaik we don't have a way to go from internal metrics.Factory to OTEL API. We would need to extend the Service (cmd/internal/flags) to initialize OTEL SDK. Main reason I didn't push for that before is that OTEL SDK performance sucks compared to Prometheus SDK, since OTEL does not support bound instrument and we're using them everywhere.